Libraries/WebSocket/RCTWebSocketExecutor.m

```diff
@@ -49,9 +52,25 @@ - (instancetype)initWithURL:(NSURL *)URL
 - (void)setUp
 {
   if (!_url) {
+    NSString *host = @"localhost";
+    NSInteger port = 8081;
+
+    NSURL *bundleURL = self.bridge.bundleURL;
+    if ([bundleURL.scheme hasPrefix:@"http"]) {
+      host = bundleURL.host;
+      port = [bundleURL.port integerValue];
+    }
```

Comment:
@nicklockwood I think we've talked about reusing `bundleURL` in `RCTWebSocketExecutor` to reduce amount of URL configurations needed, because in most cases the debugger is running on the same machine as the packager. What do you think about the proposed solution?

Side note: this will fail to guess the debugger URL when loading from file, but we already have tons of problems with debugging offline files: Chrome doesn't support loading from `file:///` protocol (without additional flags), we don't store sourcemaps by default, there is no ability to load file from remote device, etc.